# Characterizing a century of genetic diversity and contemporary antigenic diversity of N1 neuraminidase in IAV from North American swine

Graphics and analysis files for the analysis of the N2.2002 neuraminidase from 2009-2018 (swine) in the manuscript:

David E. Hufnagel, Katharine M. Young, Zebulun Arendsee, L. Claire Gay, C. Joaquin Caceres, Daniela S. Raj√£o, Daniel R. Perez, Amy L. Vincent Baker, Tavis K. Anderson. Characterizing a century of genetic diversity and contemporary antigenic diversity of N1 neuraminidase in IAV from North American swine. bioRxiv preprint.

ABSTRACT:
Influenza A viruses (IAV) of the H1N1 classical swine lineage became endemic in North American swine following the 1918 pandemic. Additional human-to-swine transmission events after 1918, and a spillover of H1 viruses from wild birds in Europe, potentiated a rapid increase in genomic diversity via reassortment between introductions and the endemic classical swine lineage. To determine mechanisms affecting reassortment and evolution, we conducted a phylogenetic analysis of N1 and paired HA swine IAV genes in North America between 1930 and 2020. We described fourteen N1 clades within the N1 Eurasian avian lineage (including the N1 pandemic clade) and the N1 classical swine lineage. Seven N1 genetic clades had evidence for contemporary circulation. To assess antigenic drift associated with N1 genetic diversity, we generated a panel of representative swine N1 antisera and quantified the antigenic distance between wild-type viruses using enzyme-linked lectin assays and antigenic cartography. Within the N1 lineage, antigenic similarity was variable and reflected shared evolutionary history. Sustained circulation and evolution of N1 genes in swine had resulted in significant antigenic distance between the N1 pandemic clade and classical swine lineage. We also observed a significant increase in the rate of evolution in the N1 pandemic clade relative to the classical lineage. Between 2010 and 2020, N1 clades and N1-HA pairings fluctuated in detection frequency across North America, with hotspots of diversity generally appearing and disappearing within two years. We also identified frequent N1-HA reassortment events (n = 36), which were rarely sustained (n = 6) and sometimes also concomitant with the emergence of new N1 genetic clades (n = 3). These data form a baseline from which we can identify N1 clades that expand in range or genetic diversity that may impact viral phenotypes or vaccine immunity and subsequently the health of North American swine.
